include/exadg/incompressible_navier_stokes/spatial_discretization/spatial_operator_base.cpp

```diff
@@ -249,20 +249,87 @@ template<int dim, typename Number>
 void
 SpatialOperatorBase<dim, Number>::distribute_dofs()
 {
+  if(param.spatial_discretization == SpatialDiscretization::L2)
+  {
+    fe_u = std::make_shared<dealii::FESystem<dim>>(dealii::FE_DGQ<dim>(param.degree_u), dim);
+
+    pcout << std::endl
+          << "Discontinuous Galerkin finite element discretization:" << std::endl
+          << std::endl
+          << std::flush;
+  }
+  else if(param.spatial_discretization == SpatialDiscretization::HDIV)
+  {
+    // In-parameter for RT is the degree in tangent direction, i.e the degree in normal direction
+    // will be degree+1. The FE also saves fe_u.degree = degree+1.
+    // degree - 1 is used so that the same Gauss quadrature can be used for exact integration.
+    fe_u = std::make_shared<dealii::FE_RaviartThomasNodal<dim>>(param.degree_u - 1);
+
+    pcout << std::endl
+          << "Hdiv-conforming Raviart-Thomas finite element discretization:" << std::endl
+          << "(Note that the given degree for the velocity is the degree in normal direction."
+          << std::endl
+          << " The degree in tangent direction is degree - 1. Note also that the number of dofs"
```

Comment:
This is hard to understand: With the wording "Note that the given degree", do you refer to the input parameter ``param.degree_u`` or to the constructor argument of the finite element? In my opinion, this does not become clear.

To be more precise, print explicitly the degree in normal direction and the degree in tangential direction to the screen:

```C++
std::cout << "Degree in normal direction: " << param.degree_u << std::endl;
std::cout << "Degree in tangential direction: " << param.degree_u - 1 << std::endl;
```

In my opinion, this is the relevant information for the user. A user does not need to know how the constructor is defined in deal.II. The user wants to know the degrees in normal/tangential direction.

Reply:
I think the comment that `degree_u` is the degree in normal direction in case of Hdiv + RT should be written here
https://github.com/exadg/exadg/blob/8382b8bacb87b907111ffe60e9cc48d7e86c2a4c/include/exadg/incompressible_navier_stokes/user_interface/parameters.h#L283

The output to cout could/should also be done in `Parameters::print()` (e.g. introduce an if-else for the output depending on L2 vs. HDIV. Then,  a comment in the spatial discretization operator, namely that the constructor argument of dealii::FE takes the tangential degree, should be enough.